modules/apps/27-interchain-accounts/module.go

```diff
@@ -224,8 +223,3 @@ func (AppModule) ProposalMsgs(simState module.SimulationState) []simtypes.Weight
 func (AppModule) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation {
 	return nil
 }
-
-// RegisterStoreDecoder registers a decoder for interchain accounts module's types
-func (AppModule) RegisterStoreDecoder(sdr simtypes.StoreDecoderRegistry) {
-	sdr[types.StoreKey] = simulation.NewDecodeStore()
```

Comment:
We can no-op or use:

```go
sdr[controllertypes.StoreKey] = simulation.NewDecodeStore()
sdr[hosttypes.StoreKey] = simulation.NewDecodeStore()
```

Is it maybe better to register the submodule stores in case they are needed in future? Otherwise happy to keep this empty with no-op.

Reply:
I feel indifferent on whether its using controller/host or nooping!